pkg/webhook/storageclass.go

```diff
@@ -165,11 +109,16 @@ func applyV1StorageClassPatch(sc *storagev1.StorageClass) *v1.AdmissionResponse
 		return reviewResponse
 	}
 
-	if _, ok := sc.Parameters["instancePrefix"]; ok {
```

Comment:
in line 107 we can use strings.ToLower to normalize and check only against "false"

Reply:
ack